x-pack/plugins/reporting/public/management/index.ts

```diff
@@ -5,7 +5,25 @@
  * 2.0.
  */
 
-export { ReportErrorButton } from './report_error_button';
-export { ReportDeleteButton } from './report_delete_button';
-export { ReportDownloadButton } from './report_download_button';
-export { ReportInfoButton } from './report_info_button';
+import { InjectedIntl } from '@kbn/i18n/react';
+import { ApplicationStart, ToastsSetup } from 'src/core/public';
+import { LicensingPluginSetup } from '../../../licensing/public';
+import { UseIlmPolicyStatusReturn } from '../lib/ilm_policy_status_context';
+import { ReportingAPIClient } from '../lib/reporting_api_client';
+import { ClientConfigType } from '../plugin';
+import type { SharePluginSetup } from '../shared_imports';
+
+export interface ListingProps {
```

Comment:
Making this interface shared since a lot of components that are downstream of `ReportListing` receive the same props as their parent.